### DISCUSSION TITLE
Feature request: Customize Datepicker’s range symbol into input

### DISCUSSION THREAD

**Discussion by eugene-lazarev** [primefaces/community#2846] [PrimeVue]
Hi there!

It would be great to have an option to change default symbols space + dash + space (` - `) between two dates in input while range is choosen (see screenshot below).

It would be very nice to have this option because, for example, in Russian we prefer to use n-dash (without spaces) for ranges.

Thanks!

<img width="659" alt="image" src="https://github.com/user-attachments/assets/79969b91-ac1e-41de-90f2-d6df46575cd6">

**Comment by j0rgedev** [comment]
Hey @eugene-lazarev, I've already reached out to the PrimeVue team to ask about this feature through the PrimeVue contributors forum on Discord and whether it aligns with their plans. I think it's a great suggestion, especially for improving localization. I’m also interested in contributing to this feature and exploring how it could be implemented. I'll keep you updated once I hear back from them

**Reply by eugene-lazarev** [reply]
Hey, do you have any updates for last week about this?

**Reply by j0rgedev** [reply]
Hello @eugene-lazarev. Unfortunately, no one from the team has responded me, so I guess @cagataycivici could help us with this. Apologies for tagging you 🙏🏻
